<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyBoardForm</class>
 <widget class="QWidget" name="KeyBoardForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_A">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_B">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_C">
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_D">
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_E">
       <property name="text">
        <string>E</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_F">
       <property name="text">
        <string>F</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_G">
       <property name="text">
        <string>G</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_H">
       <property name="text">
        <string>H</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_I">
       <property name="text">
        <string>I</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_J">
       <property name="text">
        <string>J</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_K">
       <property name="text">
        <string>K</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_L">
       <property name="text">
        <string>L</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_M">
       <property name="text">
        <string>M</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_N">
       <property name="text">
        <string>N</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_O">
       <property name="text">
        <string>O</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_P">
       <property name="text">
        <string>P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_Q">
       <property name="text">
        <string>Q</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_R">
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_S">
       <property name="text">
        <string>S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_T">
       <property name="text">
        <string>T</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_U">
       <property name="text">
        <string>U</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_V">
       <property name="text">
        <string>V</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_W">
       <property name="text">
        <string>W</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_X">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_Y">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_Z">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_GANG">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_XIE">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_DOU">
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_POINT">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_1">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_4">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_5">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_6">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_7">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_8">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_9">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_0">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_CAP">
       <property name="text">
        <string>大/小写</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_BANK">
       <property name="text">
        <string>空格</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeyBoardBtn" name="pushButton_DEL">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QKeyBoardBtn</class>
   <extends>QPushButton</extends>
   <header location="global">qkeyboardbtn.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
